packages/hash/api/src/graphql/typeDefs/knowledge/link.typedef.ts

```diff
@@ -25,11 +25,11 @@ export const persistedLinkTypedef = gql`
     """
     The link's source entity.
     """
-    sourceEntity: PersistedEntity!
+    sourceEntity: UnknownPersistedEntity!
     """
     The link's target entity.
     """
-    targetEntity: PersistedEntity!
+    targetEntity: UnknownPersistedEntity!
```

Comment:
This fixed a type error I was getting when trying to obtain the `targetEntity` using a `PersistedEntityFields` fragment in the query. I think this may be related to the issue discovered by Yusuf earlier this week, tracked in [this task](https://app.asana.com/0/1203007126736604/1203132456856955/f). I'll further investigate what's going on in that task.

Reply:
Was about to ask, thanks for highlighting